### PR TITLE
[MWPW-151474] Ensure carousel navigation is displayed after everything has been loaded

### PR DESCRIPTION
**Pull request by narcis-radu**
Fixes a CLS issue caused by the carousel controls. It took me multiple attempts to reproduce, but I saved the state [here](https://pagespeed.web.dev/analysis/https-main--cc--adobecom-hlx-live-products-firefly-features-ai-painting-generator/2jogkf78pc?form_factor=mobile). The issue is more often visible on the [carousel demo page](https://pagespeed.web.dev/analysis/https-main--milo--adobecom-hlx-live-docs-library-blocks-carousel/i5pv7gou7a?form_factor=mobile).

Resolves: [MWPW-151474](https://jira.corp.adobe.com/browse/MWPW-151474)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.live/products/firefly/features/ai-painting-generator?martech=off&georouting=off
- After: https://main--cc--adobecom.hlx.live/products/firefly/features/ai-painting-generator?martech=off&georouting=off&milolibs=carousel-cls--milo--narcis-radu
- After: https://carousel-cls--milo--narcis-radu.hlx.live/docs/library/blocks/carousel?martech=off&georouting=off

### CODE REVIEW

**Review by overmyheadandbody — COMMENTED** [review]
I can't say I'm a huge fan of having the same class in multiple places that represent different elements. Could we either:

1. change the class name to something that is obvious is a modifier and has no end styling associated? OR
2. wrap those navigation elements in a wrapper to begin with and execute the logic just on that element?

Otherwise, congrats on isolating the issue, I know it was very tough to catch!

**Comment by narcis-radu** [comment]
@overmyheadandbody - thanks for the idea, I have made the changes.

**Comment by narcis-radu** [comment]
Unit tests failure is fixed by another PR https://github.com/adobecom/milo/pull/2426

**Review by mokimo — APPROVED** [review]

File: libs/blocks/carousel/carousel.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function decorateNextPreviousBtns() {
   const previousBtn = createTag(
     'button',
     {
-      class: 'carousel-button carousel-previous',
+      class: 'carousel-button carousel-previous is-delayed',
```
Comment (overmyheadandbody) [inline libs/blocks/carousel/carousel.js:23]: There's a lot of repetition for `is-delayed`, would it be worth to turn it into a constant?

**Review by overmyheadandbody — APPROVED** [review]
Thanks for looking into the changes!

**Review by robert-bogos — APPROVED** [review]

**Review by SilviuLCF — APPROVED** [review]
Verified , ready to STAGE
Test details https://jira.corp.adobe.com/browse/MWPW-151474

**Pull request merged**